lib/transbank/sdk/onepay/validators/amount_validator.rb

```diff
@@ -0,0 +1,23 @@
+module Transbank
+  module Onepay
+    module Validators
+      class AmountValidator
+        include Transbank::Onepay::Errors
+
+        # @raise [InvalidAmountError] When amount is not numeric nor integer
+        def self.validate(amount, nullable=false)
+          if amount.nil? && nullable
+            return
+          end
+          float_amount = Float(amount)
+        rescue ArgumentError, TypeError
```

Comment:
I don't know about ruby. but, is this okay? indentation on lines 13 and 15

Reply:
[Here](https://github.com/rubocop-hq/ruby-style-guide#implicit-begin) you can see this style in the "good" block

```ruby
# good
def foo
  # main logic goes here
rescue
  # failure handling goes here
end
```